### PR TITLE
add standard github actions

### PR DESCRIPTION
**Pull request by Krastanov**

### CODE REVIEW

**Comment by Krastanov** [comment]
copied from Graphs.jl

**Comment by Krastanov** [comment]
All of this is copied from Graphs.jl. I did not put any link to docs. Should be straightforward to merge (especially if organization secrets for codecov are already set)

**Comment by Krastanov** [comment]
@gdalle  I will merge these in a couple of days and do a few more maintenance/cleanup tasks unrelated to modifying the actual code (making sure tests run, having aqua and jet, etc). For each such task I will wait a couple of days in case someone wants to approach it differently, but I will not wait for a formal review given that it is not touching actual code. If that is against community guidelines I am happy to do it differently.

**Comment by simonschoelly** [comment]
Why not merge it now and then do other PRs? In my experience, smaller PRs get merged faster.

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (simonschoelly) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]:
```suggestion
 - output from `] status Graphs` surrounded by backticks (``)
  - output from `] status GraphsMatching.jl` surrounded by backticks (``)
```

**Comment by Krastanov** [comment]
I am the one that created the PR. I do not want to merge it immediately in case someone wants to dissent. But yes, indeed, other changes will be in separate PRs -- I did not mean to imply differently.

**Review by simonschoelly — APPROVED** [review]
Looks good from my side - if there is something not working we might see it after we have merged it to master.

**Comment by simonschoelly** [comment]
I noticed that we did not have branch protection enabled for the master branch so I did that now. From now on, all PR's require one approval before they can be merged.

**Comment by Krastanov** [comment]
Ah, noted! That is what I meant when I wrote "If that is against community guidelines I am happy to do it differently."

There was a minor typo that is now fixed. I will squash and merge.

**Pull request merged**
